src/pages-and-resources/xpert-unit-summary/data/api.js

```diff
@@ -0,0 +1,22 @@
+import { getConfig } from '@edx/frontend-platform';
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+
+function getXpertSettingsUrl(courseId) {
+  return `${getConfig().LMS_BASE_URL}/ai_aside/v1/${courseId}`;
```

Comment:
I think we should get ai-aside installed into the CMS so this could use the CMS URL, it'd just make more sense

Reply:
Now that we have ai-aside properly set up as a studio plugin I was able to run a successful local test changing LMS_BASE_URL to STUDIO_BASE_URL so that is ready to go

Reply:
This has been changed